code/messaging/src/phone/java/com/adobe/marketing/mobile/MessagingConstant.java

```diff
@@ -99,17 +101,62 @@ private XDMDataKeys() {}
                 private PushNotificationDetailsDataKeys() {}
             }
         }
+
+        static final class Offers {
+            static final String PROPOSITIONS = "propositions";
+            static final String DECISION_SCOPES = "decisionscopes";
+            static final String ACTIVITY_ID = "activityId";
+            static final String PLACEMENT_ID = "placementId";
+            static final String ITEM_COUNT = "itemCount";
+            static final String ERROR = "error";
+            static final String TYPE = "type";
+            static final String PERSONALIZATION_DECISIONS = "personalization:decisions";
+            static final String PREFETCH = "prefetch";
+            static final String RETRIEVE = "retrieve";
+            static final String REQUEST_EVENT_ID = "requestEventId";
+            static final String XDM_QUERY = "query";
+            static final String XDM = "xdm";
+            static final String DATA = "data";
+            static final String DATASET_ID = "datasetId";
+            static final String XDM_EVENT_TYPE = "eventType";
+            static final String PERSONALIZATION_REQUEST = "personalization.request";
+            static final String PAYLOAD = "payload";
+            static final String ACTIVITY = "activity";
+            static final String PLACEMENT = "placement";
+            static final String ID = "id";
+            static final String ITEMS = "items";
+            static final String CONTENT = "content";
+
+            private Offers() {}
+        }
```

Comment:
i don't think we end up needing all of these - let's remember when we get closer to release to review these for use